src/tests/functional/shared_test_classes/src/single_layer/dft.cpp

```diff
@@ -22,26 +34,26 @@ std::string DFTLayerTest::getTestCaseName(const testing::TestParamInfo<DFTParams
     result << "SignalSize=" << CommonTestUtils::vec2str(signalSize) << "_";
     result << "Inverse=" << (opType == ngraph::helpers::DFTOpType::INVERSE) << "_";
     result << "TargetDevice=" << targetDevice;
-    return result.str();
+    auto str = result.str();
+    replace_if(str.begin(), str.end(), invalidTestSuffixChar, '_');
+    return str;
 }
 
 void DFTLayerTest::SetUp() {
-    InferenceEngine::SizeVector inputShapes;
+    InferenceEngine::SizeVector inputShape;
     InferenceEngine::Precision inputPrecision;
     std::vector<int64_t> axes;
     std::vector<int64_t> signalSize;
     ngraph::helpers::DFTOpType opType;
-    std::tie(inputShapes, inputPrecision, axes, signalSize, opType, targetDevice) = this->GetParam();
+    tie(inputShape, inputPrecision, axes, signalSize, opType, targetDevice) = GetParam();
     auto inType = FuncTestUtils::PrecisionUtils::convertIE2nGraphPrc(inputPrecision);
-    ngraph::ParameterVector paramVector;
-    auto paramData = std::make_shared<ngraph::opset1::Parameter>(inType, ngraph::Shape(inputShapes));
-    paramVector.push_back(paramData);
-
-    auto paramOuts = ngraph::helpers::convert2OutputVector(ngraph::helpers::castOps2Nodes<ngraph::op::Parameter>(paramVector));
-    auto dft = ngraph::builder::makeDFT(paramOuts[0], axes, signalSize, opType);
-
+    ngraph::ParameterVector params { std::make_shared<ngraph::opset1::Parameter>(inType, ov::Shape { inputShape }) };
+    auto& data = params.front();
+    data->set_friendly_name("data");
+    auto dft = ngraph::builder::makeDFT(data, axes, signalSize, opType);
 
     ngraph::ResultVector results{std::make_shared<ngraph::opset1::Result>(dft)};
-    function = std::make_shared<ngraph::Function>(results, paramVector, "DFT");
+    function = std::make_shared<ngraph::Function>(results, params, "DFT");
+    threshold = 0.05;
```

Comment:
where is this used?

Reply:
This is a member of LayerTestsCommon base class. We need this because the default 1e-2f threshold is not enough for these tests.

Reply:
Could you explain why the accuracy could not meet?

Reply:
This can be because ngraph reference implementation use FFT as (I)DFT implementation and these methods can have different precisions.